bitnami/kube-prometheus/templates/exporters/core-dns/service.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if and .Values.coreDns.enabled .Values.coreDns.service.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "kube-prometheus.fullname" . }}-coredns
+  namespace: kube-system
```

Comment:
```suggestion
  namespace: {{ .Values.coreDns.namespace }}
```

Reply:
Good point!
I prepared the values but forgot to make change here...
Corrected already.